Fix: align table items in screens: Deposits, Transactions, and Disputes

htdat opened the pull request:
Fixes #2351

#### Changes proposed in this Pull Request

- I just simply add this property to columns that we'd aligin left

```js
			isLeftAligned: true
``` 

- Remove some CSS code that has aligned these items to `center`. 
- Update relevant snapshots. 

#### Testing instructions

1. Visit these pages under Deposits, Transactions, and Disputes, and notice their layouts.
2. Apply this PR.
3. Remember to run `npm run dev` to rebuild the React code. 
4. Visit these pages again. 
5. Expect: **Text fields should be left-aligned and numeric values should be right-aligned.** 

### Screenshots - left is the current `develop` branch, right is with this PR. 

![dep](https://user-images.githubusercontent.com/10045087/127281532-7f705730-2b7c-4002-8bc2-fb28920aea3f.png)

![trans](https://user-images.githubusercontent.com/10045087/127281541-41af6ee7-9cd2-4005-a85d-70532f083785.png)

![dis](https://user-images.githubusercontent.com/10045087/127281560-54d9d956-6e92-499b-a091-5e860fadc191.png)

<!--
Testing instructions: How should this be tested and how can a reviewer test the end-user functionality? Are there known issues that you plan to address in a future PR? Are there any side effects that readers should be aware of?
-->

<!--
Add as many details as possible to help others reproduce the issue and test the fix.
"Before / After" screenshots can also be very helpful when the change is visual.
-->

-------------------

- [x] Added changelog entry (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

naman03malhotra commented on readme.txt:
```diff
@@ -111,6 +111,7 @@ Please note that our support for the checkout block is still experimental and th
 * Add - Disable payment gateway when not in test mode and not using https or ssl checkout enforcement.
 * Fix - Improved errors handling during onboarding and page overview.
 * Update - Remove Account in the old Settings page.
+* Fix - Align table items according correctly. #2351
```
same comment as change log.

htdat replied:
Addressed here 0b241862b91b046adbbbff2b181b0005e51180c3

naman03malhotra reviewed (changes requested):
I like the approach, the use of existing prop is very neat, well done.

Though can you please check the mis alignment for the following columns, i.e 
1. Source columns for both transactions and disputes page.
2. When amount col. have currency symbol on transactions page.
<img width="1570" alt="Screenshot 2021-07-29 at 4 47 29 AM" src="https://user-images.githubusercontent.com/15019298/127408367-cce4658e-2f4a-4931-9445-7613e9daff15.png">
<img width="1102" alt="Screenshot 2021-07-29 at 4 48 17 AM" src="https://user-images.githubusercontent.com/15019298/127408360-26041b4a-e8a3-4591-a357-d16cf04245f5.png">

htdat commented on changelog.txt:
```diff
@@ -14,6 +14,7 @@
 * Add - Disable payment gateway when not in test mode and not using https or ssl checkout enforcement.
 * Fix - Improved errors handling during onboarding and page overview.
 * Update - Remove Account in the old Settings page.
+* Fix - Align table items according correctly. #2351
```
Addressed here 0b241862b91b046adbbbff2b181b0005e51180c3

htdat commented:
Addressed two issues above, and updated the changelog. 

![Markup on 2021-07-30 at 12:17:00](https://user-images.githubusercontent.com/10045087/127604728-ea03a0ba-a8a0-4b33-b125-163b1257c147.png)

![Markup on 2021-07-30 at 12:17:34](https://user-images.githubusercontent.com/10045087/127604717-d41ce9d6-4a85-44d0-bb28-ee5e79cada2b.png)

naman03malhotra commented:
@LevinMedia I am not sure about the alignment of the VISA label in the source column. Can you please check it one? IMO, naturally, it feels like it should be in the center.

naman03malhotra commented:
>Addressed two issues above, and updated the changelog.

Amount column looks good. Will wait for David's comment on source columns, apart from that we are good to go. 🎉

htdat commented:
@LevinMedia - would you mind having a look at @naman03malhotra's concern above? 
https://github.com/Automattic/woocommerce-payments/pull/2589#issuecomment-889760818

vbelolapotkov commented:
@htdat fyi, there is a changelog conflict now, could you please resolve it.

htdat commented:
> @htdat fyi, there is a changelog conflict now, could you please resolve it.

Resolved it.

LevinMedia commented:
@naman03malhotra 

> I am not sure about the alignment of the VISA label in the source column. Can you please check it once? IMO, naturally, it feels like it should be in the center.

I agree with you on the center align for the logo... could we make a case here when it's neither text nor number we center it?

htdat commented:
> I agree with you on the center align for the logo... could we make a case here when it's neither text nor number we center it?

Yes, we can do this case-by-case. Addressed here 0c6fa5e.

And here is what we have now:

![Markup on 2021-08-10 at 10:00:07](https://user-images.githubusercontent.com/10045087/128801932-8e79a512-0f67-4e46-8469-a3ad00cb9953.png)
![Markup on 2021-08-10 at 09:59:50](https://user-images.githubusercontent.com/10045087/128801936-33a1032b-af34-431b-8592-2eb99a8a2db1.png)

naman03malhotra commented on client/disputes/index.js:
```diff
@@ -36,42 +42,53 @@ const headers = [
 		key: 'source',
 		label: __( 'Source', 'woocommerce-payments' ),
 		required: true,
+		cellClassName: 'payment-source',
```
IMO, we can keep it more generic like: `is-center-aligned` so that we don't duplicate the same styling for other columns to be centrally aligned in the future.

htdat replied:
Brilliant idea! Updated 9f875e6

naman03malhotra reviewed (changes requested):
Looks, great now. I've left a comment regarding potential code redundancy which may occur in the future.

htdat commented:
> Looks, great now. I've left a comment regarding potential code redundancy which may occur in the future.

@naman03malhotra Great idea! I've just applied your suggestion. Ready for the review again.

naman03malhotra commented on changelog.txt:
```diff
@@ -3,6 +3,7 @@
 = 2.9.0 - 2021-xx-xx =
 * Add - *Early access*: allow your store to collect payments with iDEAL. Enable the feature in settings!
 * Add - Split discount line in timeline into variable fee and fixed fee.
+* Fix - Align table items according correctly.
```
```suggestion
* Fix - Align table items according to design correctly.
```

naman03malhotra commented on readme.txt:
```diff
@@ -101,6 +101,7 @@ Please note that our support for the checkout block is still experimental and th
 = 2.9.0 - 2021-xx-xx =
 * Add - *Early access*: allow your store to collect payments with iDEAL. Enable the feature in settings!
 * Add - Split discount line in timeline into variable fee and fixed fee.
+* Fix - Align table items according correctly.
```
Same comment as changelog

naman03malhotra reviewed (approved):
LGTM :shipit: ! left a small comment regarding changelog.
